source/_patterns/02-organisms/00-global/01-footer.mustache

```diff
@@ -1,83 +1,241 @@
+				</li>
+			</ul>
+		</div>
+
+		<div class="footer-accordian-group" id="get-to-know-us_accordion">
+			<a href="#footer-menu-get-to-know-us" data-kv-accordion aria-controls="footer-menu-get-to-know-us" aria-expanded="false" class="section-title">Get to know us <svg class="icon icon-down-chevron-sm"><use xlink:href="#icon-down-chevron-sm" /></svg></a>
+			<ul id="footer-menu-get-to-know-us" aria-hidden="true">
+				<li class="footer-link-about">
+					<a href="{{about_href}}" kvtrackevent="Footer|click-Get to know us-About us">About</a>
+				</li>
+				<li class="footer-link-how-kiva-works">
+					<a href="{{how_kiva_works_href}}" class="how_kiva_works_href" kvtrackevent="Footer|click-Get to know us-How Kiva works">How Kiva works</a>
+				</li>
+				<li class="footer-link-faqs">
+					<a href="{{faq_href}}" class="faq_href" kvtrackevent="Footer|click-Get to know us-FAQs">FAQs</a>
+				</li>
+				<li class="footer-link-where-kiva-works">
+					<a href="{{where_kiva_works_href}}" class="where_kiva_works_href" kvtrackevent="Footer|click-Get to know us-Where Kiva works">Where Kiva works</a>
+				</li>
+				<li class="footer-link-blog">
+					<a href="{{blog_href}}" class="blog_href" kvtrackevent="Footer|click-Get to know us-Blog">Blog</a>
+				</li>
+				<li class="footer-link-partner-with-us">
+					<a href="{{partner_with_us_href}}" class="partner_with_us_href" kvtrackevent="Footer|click-Get to know us-Partner with us">Partner with us</a>
+				</li>
+				<li class="footer-link-contact-us">
+					<a href="{{contact_href}}" class="contact_href" kvtrackevent="Footer|click-Get to know us-Contact us">Contact us</a>
+				</li>
+				<li class="footer-link-help">
+					<a href="{{help_href}}" kvtrackevent="Footer|click-Get to know us-Help">Help</a>
+				</li>
+			</ul>
+		</div>
+
+		<div class="footer-accordian-group" id="community_accordion">
+			<a href="#footer-menu-community" data-kv-accordion aria-controls="footer-menu-community" aria-expanded="false" class="section-title">Community <svg class="icon icon-down-chevron-sm"><use xlink:href="#icon-down-chevron-sm" /></svg></a>
+			<ul id="footer-menu-community" aria-hidden="true">
+				<li class="footer-link-teams">
+					<a href="{{teams_href}}" class="teams_href" kvtrackevent="Footer|click-Community-Teams">Teams</a>
+				</li>
+				<li class="footer-link-studens-educators">
+					<a href="{{students_href}}" class="students_href" kvtrackevent="Footer|click-Community-Students and educators">Students and educators</a>
+				</li>
+			</ul>
+		</div>
+
+		<div class="footer-accordian-group" id="explore_accordion">
+			<a href="#footer-menu-explore" data-kv-accordion aria-controls="footer-menu-explore" aria-expanded="false" class="section-title">Explore <svg class="icon icon-down-chevron-sm"><use xlink:href="#icon-down-chevron-sm" /></svg></a>
+			<ul id="footer-menu-explore" aria-hidden="true">
```

Comment:
Is there JavaScript elsewhere that sets `aria-hidden` to false?

Reply:
Yes, this is controlled by the `accordian.js` mechanisms.